plugin/plugin.go

```diff
@@ -81,7 +84,7 @@ func Exec(ctx context.Context, args Args) error {
 		environ         = toEnvironment(args)
 		environmentID   = toEnvironmentId(args)
 		environmentType = toEnvironmentType(args)
-		issues          []string
+		issues          = extractIssues(args)
```

Comment:
don't modify this and just define the type []string

Reply:
done